lib/filewatch/tail.rb

```diff
@@ -240,8 +240,24 @@ def _sincedb_write
     def quit
       _sincedb_write
       @watch.quit
+      close_files
     end # def quit
 
+    public
+    def close_file(path)
+      @watch.unwatch(path)
+      file = @files.delete(path)
+      return if file.nil?
+      _sincedb_write
```

Comment:
why is this necessary? after line 242's `@watch.quit` we shouldn't receive more notifications of new content